cmd/cluster/azure/create.go

```diff
@@ -28,6 +28,8 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const SATokenIssuerSecret = "sa-token-issuer-key"
```

Comment:
This secret name will collide in a namespace with other HCs right?  Do we want to make prepend the HC name?